apps/ca.c

```diff
@@ -82,7 +82,9 @@
 #define ENV_DATABASE            "database"
 
 /* Additional revocation information types */
+typedef int REVINFO_TYPE;
```

Comment:
Typedef to scalar types doesn't usually buy very much; since this is contained within ca.c I wonder if switching to an enum is better.
You'd probably want to explicitly specify the values (instead of relying on the behavior of the values being incremented by one, which is mandated by the standard but a bit opaque) to preserve the ordering requirement for the revinfo types and OPTION_CHOICEs, though I guess the OPTION_CHOICEs are already relying on that behavior so maybe it's not a big deal.